src/bundler/module-registry/module-cdn.ts

```diff
@@ -14,7 +15,7 @@ export interface IResolvedDependency {
   d: number;
 }
 
-const CDN_VERSION = 4;
+const CDN_VERSION = 5;
```

Comment:
Version 4 was the last one with JSON, version 5 and greater will use msgpack.

We can also change this so the endpoint to something like `/v1/package/pack/${encoded_specifier}` and `/v1/package/json/${encoded_specifier}` and than return a redirect response on the sandpack cdn

Reply:
Will this change imply any breaking changing on Sandpack? Or should this just work?

Reply:
All backwards compatible